mariner/mariner.go

```diff
@@ -97,22 +103,22 @@ func (c Config) Update() error {
 			continue
 		}
 
-		if !strings.HasPrefix(entry.Name(), "cbl-mariner-") {
+		if !(strings.HasPrefix(entry.Name(), "cbl-mariner-") || strings.HasPrefix(entry.Name(), "azurelinux-")) {
```

Comment:
We use `cbl-mariner-` and `azurelinux-` prefixes a few times.
What if we move them to constants?

Reply:
done